### ISSUE TITLE
Filter Deck crashes 

### ISSUE THREAD

**Issue by LBX1002** [ankidroid/Anki-Android#5758]
###### Reproduction Steps

1.  Tap on filtered deck.
2. Program crashes
3. 

###### Expected Result
Deck to open
(Last apk without this issue was 2.9.3.b1)

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

###### Research
*Enter an [ x ] character to confirm the points below:*

[  ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[  ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[  ] I have searched for similar existing issues here and on the user forum

**Comment by mikehardy** [comment]
Hi there! Thanks for this report, I'm tracking a couple things right now about filtered decks not working well #5708 is another example that may be similar or duplicate but I have not confirmed.

Especially thanks for the pointer to the last known working version, that can help me pinpoint what the issue is

**Comment by LBX1002** [comment]
Hi Mike. #5708 is a bit different. On my first attempt, I get kicked out of
the app and a crash report message comes which I send in. Then I reopen and
try again and it kicks me out but no report comes up.
If you need me to reinstall one of the new versions and provide any info
please let me know.

**Comment by mikehardy** [comment]
@LBX1002 what's the UUID for your app install so I can locate the crash report? https://ankidroid.org/docs/help.html

**Comment by mikehardy** [comment]
I am about to be traveling for 10 days in some remote islands so I won't be able to see fixes here to a release until I get back. I'm interested in helping locate the problem of course but I retargeted it for a 2.9.5 release so I can do 2.9.4 tomorrow before I go - it has fixes for about 90% of our crashes now so will help lots and I don't want to hold it up

**Comment by LBX1002** [comment]
Understood. Wish you safe travels.

**Comment by LBX1002** [comment]
2.9.4 works fine.  Thanks.

**Comment by MuhamHS** [comment]
Hello @LBX1002 

How did you change your version to 2.9.4?
I checked the google play and it shows me the new functions (from 18.02.2020), but there is no button for an update to this version.

**Comment by LBX1002** [comment]
I downloaded it from the alpha/beta versions site.

**Comment by MuhamHS** [comment]
Thank you !

**Comment by mikehardy** [comment]
@LBX1002 / @MuhamHS I'm back and looking through things. Sounds like this might be working for you now? If so that's fantastic, and I'll close this, please let me know. Thanks!

**Comment by MuhamHS** [comment]
Hi MikeHardy, welcome back!

Unfortunately, my problem is still there. I can't say anything about the neweset Anki Version (2.9.4.). I downloaded this version from your website, but everytime I want to install it, I receive the message "App wurde nicht installiert" ("app has not been installed"), without any further explanation whatsoever. 
Some resources say that it may be because Android is not compatibel with an app or some default setting, that forbids to download unknown apps.. But before installing there is the message "App benötigt keine besonderen Zugriffsrechte" ("app doesn't need special access rights?"), which is contraditory

That's why I wanted to wait for 2.9.5., before experimenting any further.

You said you have reproduced my deck. Does it work now, with 2.9.4?

**Comment by mikehardy** [comment]
@MuhamHS thanks, glad to be back :-)

I have fixed this *I think* in #5757 (linked in to #5708 which is your issue I think and has more information).

You must be able to reliably switch versions to test things or I will not be able to help. I can say that 2.9.4 is out on the stable channels available from the play store (so maybe you can just uninstall AnkiDroid then install it via official means). Installing directly will require you to allow to install things from unknown sources yes, you can search the general internet for advice on that. I want to help but you have to partner with me for it to be effective

Still curious to hear @LBX1002 experience

**Comment by MuhamHS** [comment]
I understand and sorry for my lack of cooperation.

I just thought 2.9.5 is coming up soon anyway, that's why I wanted to wait before experimenting.(the libre office thing from the other thread is bothering me until today, I can't switch back to open office as standard, even after uninstalling libre office. This is another issue and doesn't have to do anything with you, but just to explain, why I hate experimenting on my own too much, especially when it didn't add to solution at all, just to be stuck with whole new problems)

Anyway, I will try your advice and come back :)

**Comment by MuhamHS** [comment]
I uninstalled and installed Anki, but the problem is still there. The custom deck crashes. 
It is the exakt same problem I described in #5708 

It's this problem:

Let's take for example the deck „A Little Life“ with 345 cards.
Right now it shows „0 0 0“, because I finished studying this deck.

If I use „study in advance“ and choose only „1 day in advance“, it says „congratulations, you are finished for today.“, and I don't get to study this deck. Same for 10 days, 50 days, 100 days, 150 days.. (I tried them out)

Until I choose 200 days in advance, then it says, I have to study 3 cards.
If I choose 250 days, it says, I have to study 5 cards.
If I choose 300 days, it says, I have to study 19 cards.
If I choose 333 days, it says, I have to study 52 cards.

And so on.. Now for this specific deck, the „333 days in advance“ doesn't work to study the whole deck. I have to choose „555 days in advance“, in order to study the whole deck.

No matter which option I take (250 days or 300 days..), the moment the first card appears and I click to see the answer, I receive the message „Sammlung nicht geöffnet“ („collection not opened“) and that something is wrong with the deck or there might be a bug etc..
